EIPS/apply-coinbase-rewards-at-end-of-block.md

```diff
@@ -0,0 +1,139 @@
+---
+eip: *
```

Comment:
```suggestion
eip: 7541
```
Assigning next sequential EIP/ERC number.
EIP/ERC numbering changed to sequential from 7500 and is no longer the PR number.

Please also update the filename.

Reply:
done!